lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialService.java

```diff
@@ -112,14 +114,24 @@ private Name generateName(PersonIdentity personIdentity) {
 
     private Object[] calculateEvidence(KBVItem kbvItem) {
 
-        /// TODO: Handle multiple evidence items
-
         Evidence evidence = new Evidence();
         evidence.setType(EvidenceType.IDENTITY_CHECK);
         evidence.setTxn(kbvItem.getAuthRefNo());
 
-        /// TODO: calculate evidence score
-        evidence.setVerificationScore(2);
+        // Just in case we're sent null data, we don't want to try and do an upper case on it
+        if (kbvItem.getStatus() == null) {
```

Comment:
If we don't have a status then this can be considered as an exceptional circumstance and we can throw an exception here instead